tests/test_heroku.py

```diff
@@ -136,6 +136,24 @@ def test_sanity_check_raises_on_invalid_config_combo(self, heroku, stub_config):
             heroku.sanity_check(stub_config)
         assert excinfo.match("dyno type not compatible")
 
+    def test_sanity_check_raises_on_invalid_web_dyno_combo(self, heroku, stub_config):
```

Comment:
These maybe cry out for a parameterized test, but we don't do much of that so I'm fine with it as-is.

Reply:
For sure, but I think this little thing isn't the place to start refactoring our test structures.